### ISSUE TITLE
FEATURE: enable date formating on Pods date field

### ISSUE THREAD

**Issue by davedenis** [pods-framework/pods-beaver-builder-themer-add-on#40]
While it's possible to format the date for the built in date field, for example:
[wpbb post:date format="M j, Y"]

The following does not work:
[wpbb post:pods_display field="todays_date" format="M j, Y"]

A custom function can be written and the field called via a magic tag, but allowing format on the pods_display field would be a nice enhancement to this integration with Themer.

**Comment by quasel** [comment]
Per default you can configure that within pods - but it might be an idea to divert from "default" settings on occasion! 

![ohne titel](https://user-images.githubusercontent.com/9464217/30445654-f799d6b6-9986-11e7-9cf7-352dc89c0f04.png)

**Comment by davedenis** [comment]
Yes, since parts of a CPT entry could be syndicated and re-used on various parts of the site, being able to filter them in Themer is very valuable.

Unrelated, but I don't see "Date format type" in my field. What's that?

**Comment by quasel** [comment]
That's new with 2.7 previously you could just select a format :D

**Comment by XAAANEk** [comment]
Nearly 5 years later and I would love to see this as well. Hope y'all make the decision to implement it!
